django/contrib/admin/views/main.py

```diff
@@ -275,6 +276,14 @@ def get_filters(self, request):
             raise IncorrectLookupParameters(e) from e
 
     def get_query_string(self, new_params=None, remove=None):
+        from django.utils.http import urlencode
+
+        warnings.warn(
```

Comment:
We need docs for the deprecation, in the release notes and in `docs/internals/deprecation.txt` :+1: